core/src/com/fallenflame/game/FogController.java

```diff
@@ -148,8 +149,8 @@ public void updateFogAndDraw(GameCanvas canvas, Vector2 scale, float delta) {
                                 for (int i = 0; i < ((levelModel.hasEnemy(x, y) ? levelModel.hasLessFog(x, y) ? NUM_FOG_SHOOTER : NUM_FOG_ENEMIES : NUM_FOG_NORMAL)); i++) {
                                     ParticleEffectPool.PooledEffect effect = fogPool.obtain();
                                     effect.reset();
-                                    float incX = levelModel.hasEnemy(x, y) ? (float) ((Math.random() - 0.5) * NUM_FOG_AROUND_ENEMIES) : 0;
-                                    float incY = levelModel.hasEnemy(x, y) ? (float) ((Math.random() - 0.5) * NUM_FOG_AROUND_ENEMIES) : 0;
+                                    float incX = levelModel.hasEnemy(x, y) ? (float) ((Math.random() - 0.5) * (levelModel.hasLessFog(x, y) ? NUM_FOG_AROUND_SHOOTERS : NUM_FOG_AROUND_ENEMIES)) : 0;
+                                    float incY = levelModel.hasEnemy(x, y) ? (float) ((Math.random() - 0.5) * (levelModel.hasLessFog(x, y) ? NUM_FOG_AROUND_SHOOTERS : NUM_FOG_AROUND_ENEMIES)) : 0;
```

Comment:
Lol I'm not going to pretend to understand why hasLessFog() is really isShooter()